### PR TITLE
Final adjustments

### PR DESCRIPTION
**Pull request by 4r7if3x**

### CODE REVIEW

File: resources/packaging/linux/sniffnet.desktop [hunk 1]
```diff
@@ -3,8 +3,8 @@ Version=1.0
 Type=Application
 Name=Sniffnet
 Comment=Application to comfortably monitor your network traffic
-Categories=Network;
+Categories=Network;Utility;
 Icon=sniffnet
-Exec=/usr/bin/sniffnet
```
Comment (GyulyVGC) [inline resources/packaging/linux/sniffnet.desktop:8]: what was the problem with this one?

Comment (4r7if3x) [reply resources/packaging/linux/sniffnet.desktop:8]: Nothing, but `/usr/bin` is already in system `PATH`...

P.S. I'll send a new PR, my previous commit missed a thing and this PR is again not updated automatically...

Comment (4r7if3x) [reply resources/packaging/linux/sniffnet.desktop:8]: Yes please & No, it'd be fine.

Comment (GyulyVGC) [reply resources/packaging/linux/sniffnet.desktop:8]: If someone install the app also via cargo or homebrew... which version will open just typing Sniffnet? If no problem was caused, I'd leave thing as they were

**Pull request closed without merging**